data/remnant/remnant 1 introduction.txt

```diff
@@ -3823,9 +3823,9 @@ mission "Remnant: Logistics 1"
 		has "Remnant: Defense 1: done"
 	on offer
 		conversation
-			`You step out of the <ship> amidst the strange sight of the <origin> spaceport. The structures that surround the landing pads look as much grown as built, and they seem to merge into the landing pads themselves. As usual, the dock entrance is busy. When your commlink vibrates, you have to find a place where the traffic eddies so you can step aside. When you do, you learn you need to check in with the information desk.`
+			`You step into the strange sight of the <origin> spaceport. The structures that surround the landing pads look as much grown as built, and they seem to merge into the landing pads themselves. As usual, the dock entrance is busy. When your commlink vibrates, you have to find a place where the traffic eddies so you can step aside. When you do, you learn you need to check in with the information desk.`
```

Comment:
What is the reason for this change? It doesn't seem to be related to the change you discussed in the description.

Reply:
Sorry, I should have edited the PR description. It's based on this suggestion from Warp: https://github.com/endless-sky/endless-sky/pull/10746#discussion_r1835809497